3.analyze_data/README.md

```diff
@@ -0,0 +1,21 @@
+# Analyze predicted probability output
+
+In this module, we perform multiple analyses on the predicted probability data to validate the treatment (e.g., compound, CRISPR, or ORF) with the expected phenotypic class.
+To perform comparisons between treatment and the negative control, we perform these post-hoc statistical analyses:
+
+1. Mann-whitney U test
+2. T-test
+3. Dunn test 
+
+These tests compute the median to determine the directionality of significance among treatments compared to control.
```

Comment:
@MattsonCam - there is some detail here that Jenna will need help clarifying.

Can you make suggestions when you get a chance?

It's also possible that we decide it's not worth going into this much detail given that our approach is changing.

Reply:
@jenna-tomkinson in my mind, we want to eventually decide on a single comparison to make in the analysis, so we may want to wait until this becomes more concrete. For now, we could probably mention some of the analyses, although, this might become confusing as we are still narrowing down the approach